<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Conf_RTrees</class>
 <widget class="QMainWindow" name="Conf_RTrees">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>331</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>562</width>
    <height>331</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configuraci√≥n RTrees</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Depth</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Min Sample Count</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Regression Accuracy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Native Vars</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Categories</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>CV Folds</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="use_surrogates">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Use Surrogates</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="use_1se_rule">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Use 1se Rule</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="truncate_pruned_tree">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Truncate Pruned Tree</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="calc_var_importance">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Calc Var Importance</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="max_depth">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="min_sample_count">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="regression_accuracy">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>62</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximum">
     <double>999999.000000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="max_categories">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="cv_folds">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="native_vars">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Aceptar">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>230</y>
      <width>96</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Aceptar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>562</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
